src/layers/src/scenegraph-layer/scenegraph-info-modal.tsx

```diff
@@ -20,7 +20,7 @@
 
 import React from 'react';
 import styled from 'styled-components';
-import {StyledTable as Table} from 'components/common/styled-components';
+import {StyledTable as Table} from '@kepler.gl/constants';
```

Comment:
looks like the only place `StyledTable` is used is  in screnegraph-layer, you should just move the implementation in here, instead of moving it to constant